pom.xml

```diff
@@ -95,7 +95,7 @@
         <!--<affinity.version>3.2.3</affinity.version>-->
         <antlr4.version>4.9.3</antlr4.version>
         <avro.version>1.11.0</avro.version>
-        <aws.sdk.version>1.12.183</aws.sdk.version>
+        <aws.sdk.version>1.12.267</aws.sdk.version>
```

Comment:
Maybe we could move straight forward to the latest `1.12.269`?

Reply:
That's futile effort, they release a new version every day :-)

Reply:
I've chosen version used on master, simply becauase it's the most tested one by us :)